src/main/scala/is/hail/stats/SkatModel.scala

```diff
@@ -0,0 +1,211 @@
+package is.hail.stats
+
+import breeze.linalg._
+import is.hail.methods.SkatStat
+import com.sun.jna.Native
+import com.sun.jna.ptr.IntByReference
+
+import scala.math.pow
+import net.sourceforge.jdistlib.NonCentralChiSquare
+
+/** SkatModel
+  *    A class which has all the algorithms for computing the p value of the
+  *    Variance-component score statistic thats passed into the class.
+  *  Input:
+  *    qstat                   Double
+  *      Variance-component score statistic at which to evaluate the cumulative
+  *      distribution.
+  *  Methods:
```

Comment:
after looking around, I'm not seeing very much documentation like I wrote here. Should I just delete it?